src/actions/categories/getCategoriesAsyncThunk.ts

```diff
@@ -0,0 +1,15 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { CategoryModel } from '../../models/CategoryModel';
+
+
+
+const requestUrl = "https://localhost:7284/categories"
```

Comment:
Can you move the 'https://localhost:7284' as an environment variable? This value can be changed depending on the environment - local,dev,QA,prod.

Reply:
.env file has been added to the root project catalog